include/oneapi/tbb/detail/_config.h

```diff
@@ -327,7 +327,7 @@
 #define __TBB_CPP17_UNCAUGHT_EXCEPTIONS_PRESENT             (_MSC_VER >= 1900 || __GLIBCXX__ && __cpp_lib_uncaught_exceptions \
                                                             || _LIBCPP_VERSION >= 3700 && (!__TBB_MACOS_TARGET_VERSION || __TBB_MACOS_TARGET_VERSION >= 101200))
 
-#define __TBB_TSX_INTRINSICS_PRESENT ((__RTM__ || _MSC_VER>=1700 || __INTEL_COMPILER) && !__ANDROID__)
+#define __TBB_TSX_INTRINSICS_PRESENT ((__RTM__ || _MSC_VER>=1700 || __INTEL_COMPILER) && !__ANDROID__ && !_M_ARM64)
```

Comment:
```suggestion
#define __TBB_TSX_INTRINSICS_PRESENT ((__RTM__ || __INTEL_COMPILER || (_MSC_VER>=1700 && (__TBB_x86_64 || __TBB_x86_32))
```

Reply:
@alexey-katranov - was the `!__ANDROID__` dropped from your suggestion on purpose?

Reply:
It seems weird for me that availability of TSX instructions depends on OS. If the compiler defines RTM or it is suitable compiler on x86, TSX intrinsics should present. Maybe it was some workaround but there is no any comment, so I would remove `__ANDROID__` 
It does not fully relates to your patch, typically we refactor nearby code if something seems strange/wrong. Surely, it will not an issue if you want to remain it untouched, so feel free to return `!__ANDROID__` back. The only important thing is to limit RTM support with Microsoft Compiler within x86/x64 platform.